modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao}.
```

Comment:
maybe too obvious but, should we point the operator needs to be installed on the seed cluster and in the spoke clusters?